---
output:
  pdf_document: default
  html_document:
    df_print: paged
  word_document: default
geometry: left = 2.5cm, right = 2cm, top = 2cm, bottom = 2cm
fontsize: 11pt
header-includes:
- \usepackage{setspace}\spacing{1}
- \pagenumbering{gobble}
bibliography: Biblio_Mitaraka.bib
csl: zoosystema.csl
---

\pagenumbering{arabic}

```{r, include=FALSE}
library(knitr)
library(flextable)
library(tidyverse)
library(plyr)
c<-read.csv2("1_Datasets/Characters.csv",h=T,row.names = 1) # load morpho-anatomical characters for all the species
m<-read.csv2("1_Datasets/Specimens.csv") # load collecting data for whole dataset
```


# Earthworms (Oligochaeta: Clitellata) of the Mitaraka range (French Guiana): commented checklist with description of one genus and eighteen species new to science

# Les vers de terre (Oligochaeta: Clitellata) du Massif du Mitaraka (Guyane): liste commentée avec description d'un genre et de dix-huit espèces nouvelles pour la science


\vspace{12pt}
## Running title: Earthworms of the Mitaraka Massif

\vspace{12pt}
Thibaud DECAËNS^1a^, Marie Luise Carolina BARTZ^2,3^, Alexander FEIJOO MARTÍNEZ^4^, Arnaud GOULPEAU^1^, Emmanuel LAPIED^5^, Daniel F. MARCHÁN^1^, Marie-Eugénie MAGGIA^6^, Guillaume PAPUGA^7^, Sam W. JAMES^8^

\vspace{12pt}
^1^ CEFE, Univ Montpellier, CNRS, EPHE, IRD, Montpellier, France. E-mail: thibaud.decaens@cefe.cnrs.fr, danifermch@gmail.com

^2^ Municipal Center for Culture and Development - Organic Farming: Agriculture and Organic Production Partnership at Centre for Organic and Regenerative Agriculture, Centro Empresarial, Zona Industrial – 6060-182, Idanha-a-Nova, Portugal. E-mail: bartzmarie@gmail.com

^3^ Centre for Functional Ecology, Department of Life Sciences, University of Coimbra, Coimbra, Portugal.

^4^ Facultad de Ciencias Ambientales, Universidad Tecnológica de Pereira, A.A. 097, Pereira, Colombia. E-mail: afeijoo@utp.edu.co

^5^ Taxonomia Biodiversity Fund, 7 rue Beccaria, 72012 Paris, France. E-mail: elapied@yahoo.com

^6^ Department of Integrative Biology, University of Guelph, N1G 2W1 Guelph, Canada. E-mail: mmaggia@uoguelph.ca

^7^ AMAP, Univ Montpellier, CIRAD, CNRS, INRAE, IRD, Montpellier, France E-mail: guillaume.papuga@umontpellier.fr

^8^ Sustainable Living Department, Maharishi International University, Fairfield, IA 52557, USA E-mail: massemaj@gmail.com

^a^ Corresponding author

# ABSTRACT

Intensive sampling of earthworms in French Guiana has been carried out for several years. The first published results have highlighted a very high regional diversity, a considerable geographical turnover in the composition of communities, as well as a very high proportion of species new to science. However, the work of describing these new species had not been undertaken until now. In this work, we present the results of the sampling carried out in March 2015 during the Our Planet Reviewed terrestrial expedition in the Mitaraka Range. We used an integrative taxonomy approach combining the use of DNA barcodes and morpho--anatomical characters, which allowed the identification of 31 species, nine of which were represented in our samples only by juveniles, and four of which corresponded to already known species (*Nouraguesia parare* Csuzdi and Pavlícek, 2011, *Pontoscolex corethrurus* (Müller, 1856), *Dichogaster andina* (Cognetti de Martiis, 1904) and *Dichogaster bolaui* (Michaelsen,1891)). The remaining 18 species were new to science and are described in this work. They belong to the genera *Andiorrhinus* (one species), *Martiodrilus* (nine species), *Urobenus* (one species), *Atatina* (one species), *Neogaster* (two species), *Omodeoscolex* (three species), and a new genus of Ocnerodrilidae which we describe as *Guianodrilus* gen. nov. We then discuss the Linnaean deficit that characterizes the earthworm fauna of Guyana and the taxonomic implications of our results.

# KEY-WORDS
Earthworms, new species, *Martiodrilus*, *Andiorrhinus*, *Urobenus*, *Atatina*, *Neogaster*, *Omodeoscolex*, *Guianodrilus* gn. nov., French Guiana

# RESUME

Un échantillonnage intensif des vers de terre de Guyane a été réalisé depuis plusieurs années. Les premiers résultats publiés ont mis en évidence une diversité régionale très importante, un turnover géographique considérable dans la composition des communautés, ainsi qu’une proportion très importante d’espèces nouvelles pour la science. Cependant le travail de description de ces espèces nouvelles n’avait pas été entrepris jusqu’à maintenant. Dans ce travail, nous présentons les résultats de l’échantillonnage réalisé en mars 2015 à l’occasion de la mission terrestre de la Planète Revisitée dans le Massif du Mitaraka. Nous avons pour cela utilisé une approche de taxonomie intégrative combinant l’utilisation de codes--barres ADN et de caractères morpho--anatomiques. Nous avons ainsi identifié la présence de 31 espèces, dont neuf ne sont représentées dans nos échantillons que par des juvéniles, et quatre correspondent à des espèces d’ores et déjà connues (*Nouraguesia parare* Csuzdi and Pavlícek, 2011, *Pontoscolex corethrurus* (Müller, 1856), *Dichogaster andina* (Cognetti de Martiis, 1904) and *Dichogaster bolaui* (Michaelsen,1891)). Les 18 espèces restantes se sont révélées nouvelles pour la science et sont décrites dans ce travail. Elles appartiennent aux genres *Andiorrhinus* (une espèce), *Martiodrilus* (neuf espèces), *Urobenus* (une espèce), *Atatina* (une espèce), *Neogaster* (deux espèces), *Omodeoscolex* (trois espèces), et un nouveau genre d’Ocnerodrilidae que nous décrivons sous le nom de *Guianodrilus* gn. nov. Nous discutons ensuite le déficit linnéen qui caractérise les vers de terre de Guyane ainsi que les implications taxonomiques de nos résultats.

# MOTS CLES
Vers de terre, nouvelles espèces, *Martiodrilus*, *Andiorrhinus*, *Urobenus*, *Atatina*, *Neogaster*, *Omodeoscolex*, *Guianodrilus* gn. nov., Guyane

\newpage
# INTRODUCTION

Earthworms are recognised for the ecological functions they perform in terrestrial ecosystems and the support they provide for the production of important ecosystem services. They are defined as ecosystem engineers, capable of physically modifying their edaphic environment by altering its physico-chemical properties, thereby modifying soil fertility, plant production and the living conditions of the species with which they coexist [@lavelle_ecosystem_2016]. Despite this undeniable importance for the functioning of soils and terrestrial ecosystems, earthworms are relatively poorly known from a taxonomic point of view, particularly in little-studied and difficult-to-access tropical regions [@decaens_potential_2013]. French Guiana is emblematic of this paradox. Indeed, in the early 2000s, only about twenty named species were cited there, some of which described recently [@csuzdiNewLittleKnown2011], which was far below the diversity expected for a tropical region of this size [@brown_minhocas_2007; @pavlicek_earthworm_2012-1].

Since 2010, several sampling campaigns focussing on biodiversity assessment and including earthworms have been carried out in different parts of French Guiana. @decaens_dna_2016 published a first contribution to the knowledge of earthworm diversity in the Nouragues Nature Reserve. Using an integrative taxonomy approach based on the combined use of morphological characters and DNA barcodes, they found a total of 48 species in this locality, and suggested that up to 60 species might be present there. Although they suggested on the basis of their observations that most of the species sampled might be new to science, the authors of this work did not go as far as to formally describe these taxa. Recently, @maggia_at_2021 extended this approach to six localities and re-estimated the regional diversity to nearly 120 species. Considering the high turnover of species between localities and habitats, they suggest that the number of species present in the whole territory of French Guiana could reach several hundred. However, this estimate is based on operational taxonomic units (OTUs) delimited from DNA barcodes, which were used as species proxies without further taxonomic assignments. Generally speaking, these first articles highlight the remarkable species diversity of earthworms in French Guiana, and the need to develop an effective taxonomic approach to describe the multitude of taxa that have already been sampled or that will be discovered in future field campaigns.

In this work, we publish a list of species collected during the Our Planet Reviewed expedition in the Mitaraka range [@touroultOverviewMitarakaSurvey2018], southern French Guiana. We used the DNA barcodes of @maggia_at_2021 and performed morphological diagnoses for each of the OTUs delineated by these authors. Based on the morpho-anatomical diagnoses and molecular data, we describe *Guianodrilus* gen. nov. in the family Ocnerodrilidae, and 18 species new to science in the genera *Martiodrilus* (nine species), *Andiorrhinus* (one species), *Urobenus* (one species), *Atatina* (one species), *Guianodrilus* (one species), *Neogaster* (two species) and *Omodeoscolex* (three species). We further discuss the Linnaean deficit that characterises the earthworm fauna of French Guiana, and taxonomic implications of our results.

# MATERIAL AND METHODS

<span style="font-variant:small-caps;">Study site and earthworm sampling</span>

Earthworms were sampled in March 2015 in the Mitaraka Massif, a largely unknown and currently uninhabited area in the extreme south-western corner of French Guiana [@touroultOverviewMitarakaSurvey2018]. The region belongs to the Tumuc-Humac mountain chain, extending east in the Amapá region of Brazil and west in southern Surinam (Fig. 1A). Local landscape essentially consists in an alternance of tropical rainforest ecosystem on slope and plateau positions, periodically flooded forests in lowlands, and scattered inselbergs, i.e. isolated granitic rocky outcrops that stand above the forest cover (Fig. 2A,B). Vegetation on the inselbergs includes rocky savannahs (i.e. open formations dominated by terrestrial bromeliads; Fig. 2D,G), and transition to hiltop forests (Fig. 2C). The entire area is situated in the core of the protected Parc Amazonien de Guyane.

\newpage
![](2_Figures/Decaëns et al Fig 1 Carte_Mitaraka.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 1.</span> -- Localisation of the sampling sites in the Mitaraka Massif, French Guiana. Site codes are the same as in Table 1.

The base camp was located near the Alama river, at coordinates 2.23398° of latitude and -54.4503° of longitude, and at an altitude of 330 m asl. Around this central point, the main characteristic ecosystems of the region were sampled according to a replicated sampling design and using a standardised protocol (Fig. 1B). The list of surveyed sites, the corresponding ecosystems and their geographical location is given in Table 1.

The detailed sampling protocol is given in @maggia_at_2021. Briefly, each site consisted in a 1 ha area plot, where earthworms were collected by combining (1) digging and handsorting three blocks of soil, each 25 × 25 × 20 cm (length × width × depth), (2) digging and handsorting an additional area of 1 m^2^ to a minimum depth of 30-40 cm in an area with large earthworm casts, and (3) micro-habitat sampling by visually inspecting all available micro-habitats (such as sandy to muddy sediments of stream banks, leaf litter accumulations, decaying trunks and epiphytic soils) for a standardised period of time. In addition, a number of specimens were collected opportunistically by teams working on other taxonomic groups during the expedition. Earthworms of all life stages (adults, juveniles and cocoons) were collected and kept in 100% ethanol. Ethanol was changed after 24 h to ensure optimal fixation.

<span style="font-variant:small-caps;">DNA barcoding, OTU delimitations and phylogenetic analyses</span>

The collected specimens were first sorted according to their external morphology, and up to five specimens per sampling unit and per morpho-group were selected for DNA analysis. A small tissue fragment was taken from each of them for DNA analysis at the Canadian Centre for DNA Barcoding. Lab work followed standardized protocols of the International Barcode of Life (see @maggia_at_2021 for details). Sequences and supporting information were deposited in the Barcode of Life Datasystems (BOLD) database [@ratnasinghamBOLDBarcodeLife2007] in the dataset “Earthworms from the Mitaraka Massif, French Guiana” (DS-EWMITK, DOI:dx.doi.org/10.5883/DS-EWMITK) and were subsequently submitted to GeneBank.


```{r, include=FALSE}
synt<-read.csv2("1_Datasets/Habitats.csv", h=T)
Table<-flextable(synt) 
Table <- italic(Table, j = 1)
Table <- fontsize(Table, size = 8, part = "all")
Table <- autofit(Table, add_w = -0.05)
```
\newpage
<span style="font-variant:small-caps;">Table 1.</span> -- List of the study plots were earthworms were sampled in the Mitaraka Massif.
`r knit_print(Table)`

<span style="font-variant:small-caps;">Fig. 2.</span> (next page) -- The Mitaraka Massif: A) General view of the inselberg landscape from "Borne 1", with "Sommet en Cloche" in the foreground, followed by "Tchoukouchipan" and "Mitaraka"; B) *Astrocaryum* forest plateau near Base Camp; C) Transitional forest (TF1) on the slopes of the "Sommet en Cloche"; D) Rocky savannah (RS3) showing terrestrial bromeliad (*Pepinia* sp. ) and transitional forest (TF2) in the background; E) a specimen of *Martiodrilus (Botaria) tchoukouchipan* n. sp. in the root mat of terrestrial bromeliads in a rocky savanna (RS2); the Holotype of *Martiodrilus (Botaria) flavus* n. sp. in a transitional forest (TF2); F) detail of the terrestrial bromeliad beds in the rocky savanna RS3 (all photos © T. Decaëns).

\newpage
![](2_Figures/Decaëns et al Fig 2 panel.pdf){ width=90% }

<span style="font-variant:small-caps;">Fig. 2.</span>

\newpage
Sequences were further used to delimit operational taxonomic units (OTUs) using the Automatic Barcode Gap Discovery method (ABGD, @puillandreABGDAutomaticBarcode2012a) as described in @maggia_at_2021. We further aligned sequences with MAFFT v.7 [@katohMAFFTMultipleSequence2013] with default settings, and collapsed them into unique haplotypes in FaBox 1.61 [@villesenFaBoxOnlineToolbox2007]. The best fitting evolutionary model (GTR + I + G) was selected with jModelTest v. 2.1.3 [@darribaJModelTestMoreModels2012] by applying the Akaike information criterion (AIC; @akaikeMaximumlLikelihoodIdentification1973), and Bayesian information criterion (BIC; @schwarzEstimatingDimensionModel1978). Bayesian Inference of the phylogeny was estimated with MRBAYES v.3.1.2 [@ronquistMrBayesBayesianPhylogenetic2003] as implemented in the CIPRES Science Gateway V. 3.3. Parameters were set to 50 million generations and sampled every 5,000th generation (10,000 trees). Two independent runs each with four chains were performed and 20% of the trees were discarded as burn-in. The remaining trees were combined and summarized on a 50% majority-rule consensus tree. Separate phylogenetic trees were inferred for Rhinodrilidae plus Glossoscolecidae, Acanthodrilidae and Ocnerodrilidae, using *Hirudo medicinalis* Linnaeus, 1770 (Genbank accession HQ333519) as outgroup in each case. Uncorrected average pairwise genetic distances within and between species were calculated in MEGA 11 [@tamuraMEGA11MolecularEvolutionary2021].

<span style="font-variant:small-caps;">Morpho-anatomical descriptions and integrative taxonomy approach</span>

When available, at least three adult specimens per OTUs were selected for complete morpho-anatomical examinations, and additional specimens were considered for external morphology only and described following a similar approach to that described in @decaens_dna_2016. For external morphology, we measured body size and weight and counted the number of segments, recorded external data on setal relations (distance between setal lines), shape of prostomium, pigmentation, location and shape of clitellum, visible genital pores, modified setae, and genital markings (including tubercula pubertatis if appropriate). For internal characters, we recorded locations and form of calciferous glands, shape and size of the gizzard, muscularity of anterior septa, spermathecal characters, condition of male reproductive organs, location of the intestinal starting point, form of typhlosole (a dorsal inward extension of the intestinal wall), presence and number of caecal pouches when relevant. For the family Benhamiidae, we recorded the numbers and form of nephridia, and the characteristics of the acanthodriline male fields represented. Explanations of these morphological characters are available in @csuzdiMonographPaleotropicalBenhamiinae2010 and @righiColombianEarthworms1996. We did not consider other characters that can be used in diagnoses within certain genera (e.g. genital setae in *Martiodrilus*), as we considered that they did not provide significant information in addition to those already used in combination with molecular information. We are aware that this may be seen by some readers as a shortcoming, but given the number of new species to be described in French Guiana [@@maggia_at_2021], we adopt here a "turbotaxonomy" approach [*sensu* @butcherTurbotaxonomicStudyThai2012c] in which we try to maximise the quality of the original descriptions while trying to reduce the time cost.

We then followed the following integrative approach: 1) when two OTUs presented consistent differences in more than one morpho-anatomical character traditionally deemed of specific value, we considered them as two distinct species; 2) when two OTUs were morphologically indistinguishable, we still considered them as distinct species (in this case cryptic species) when they were clearly separated in the phylogenetic tree (i.e. did not form a monophyletic group); 3) when two OTUs were morphologically indistinguishable, and closely related in the phylogenetic tree (i.e. forming a monophyletic group), we considered them as conspecific cryptic lineages.

<span style="font-variant:small-caps;">Diversity patterns and species number estimates</span>

The species diversity at the scale of the Mitaraka range was analyzed by plotting rarefaction and extrapolation curves. We used indidence--based rarefaction curve to highlight species number accumulation as a function of the number of sampled plots within the study area, and abundance--based rarefaction curves to highlight species number accumulation as a function of the number of earthworm individual sampled within each main types of habitats and microhabitats. We also calulated the Chao index, which estimates the lower bound for the expected asymptotic species richness [@gotelliMeasuringEstimatingSpecies2013a]. All these analyses were done using the ‘iNEXT’ package for R v.3.5.3 [@hsiehINEXTINterpolationEXTrapolation2019a; @rcoreteamLanguageEnvironmentStatistical2019a].

Community--scale species richness was estimated as the comulated number of species observed at the scale of a 1 ha plot. We tested the overall habitat effect using Kruskal-Wallis rank sum test with the ‘kruskal.test’ function of R, and pair wise differences among habitats using pairwise Wilcoxon rank sum tests with the ‘pairwise.wilcox.test’ function of R. 

<span style="font-variant:small-caps;">Automated species descriptions</span>

For each of the species recognised by the integrative taxonomy approach described above, we compared the morpho-anatomical characters to those of the already known species in the genera concerned. When a species differed from the known species by at least one morpho-anatomical character of specific diagnostic value, we considered it as a species new to science and proceeded to its formal description.

We then compiled a first table containing the collection data of all the specimens collected for each species, and a second table describing the morpho-anatomical characters of each of them. We then created a script for R Markdown [@allaireRmarkdownDynamicDocuments2021a] that automatically generates the list of specimens (holotypes, paratypes and other material) and corresponding collection data, as well as a standardised description of the external morphological characters and internal anatomy of each species new to science. This automated approach allowed us to save a considerable amount of time on the description writing phase (i.e. less than one day to generate the whole manuscript). The data tables and an example of script are given in Supplementary Material.

<span style="font-variant:small-caps;">Collection abbreviations</span>

- CEFE    Centre d'Ecologie Fonctionnelle et Evolutive;

- MNHN    Muséum national d’Histoire naturelle of Paris (France)

# RESULTS

Our study produced 348 COI barcodes, which clustered into 31 OTUs belonging to the families Rhinodrilidae, Glossoscolecidae, Acanthodrilidae and Ocnerodrilidae (Fig. 3). All these OTUs are supported by bootstarp values above 99%, except for one Ocnerodilidae with had a bootstrap of 92% (Fig. 3). All of them also presented average pairwise genetic distances well above the 14% threshold usually recognised of species-level significance for earthworms [@decaens_potential_2013], even within single genus or closely related genus (Table 2,3).

Of these OTUs, nine were represented only by juveniles or fragments that could only be identified to genus or family levels (i.e. one *Andiorrhinus* sp., one *Martiodrilus* sp., one *Pontoscolex* sp., one *Rhinodrilus* sp., one *Righiodrilus* sp, one *Wegeneriona* sp. and three Ocnerodrilidae spp.), and four corresponded to already described species (*Nouraguesia parare* Csuzdi and Pavlícek, 2011, *Pontoscolex corethrurus* (Müller, 1856), *Dichogaster andina* (Cognetti de Martiis, 1904) and *Dichogaster bolaui* (Michaelsen, 1891)). The remaining 18 OTUs were found to be new to science after morpho-anatomical examination and are described herein. They belong to the genera *Andiorrhinus* (one species), *Martiodrilus* (nine species), *Urobenus* (one species), *Atatina* (one species), *Neogaster* (two species), *Omodeoscolex* (three species), and a new genus of Ocnerodrilidae described as *Guianodrilus* gen. nov.

The rarefaction curves indicate that the sampling effort deployed in our study was enough to identify a significant proportion of earthworm diversity at the scale of the study site. Indeed, the incidence--based rarefaction curve shows an inflection (Fig. 4A), and the Chao index indicates that 36 species could be present in this region of the Mitaraka montains, which is not far from the 31 species found in our study. Most of species diversity was recovered in the plateau, slope and lowland forests, whereas inselberg habitats (i.e. transition and hiltop forests, rocky savannahs) were comparatively less diversified (Fig. 4B). Regarding microhabitats, most of the species diversity was found in decaying trunks and associated termite mounds, followed by the organomineral soil (Fig. 4C).

The average community--level richness (alpha diversity) was 4.3 species observed per ha, with the highest value observed in a plateau forest where 12 species were sampled, and the lowest in a lowland forest and some inselberg habitats where only one species was found (Fig. 4D). Overall, plateau and slope forests having similarly diversified communities (6.2 species observed per ha in average), lowland forests were slightly less diversified (4.5 species observed per ha in average), and inselberg habitats showed the lowest values (2.3 species observed per ha in average). The effect of main habitat types on community--scale species richness was significant (Kruskal-Wallis test; p-value = 0.02) and was mainly explained by the lower species richness in inselberg habitats compared to plateau and slope forests (Wilcoxon rank sum test; p-values = 0.006 and 0.014, respectively). Other pairwise comparisons among habitats were not significant.

\newpage
![](2_Figures/Decaëns et al Fig 3 phylo_trees.pdf){ width=100% }
<span style="font-variant:small-caps;">Fig. 3.</span> -- Bayesian Inference of the phylogeny of species of A) Rhinodrilidae and Glossoscolecidae b) Acanthodrilidae and C) Ocnerodrilidae based on the sequence of COI. In all of them *Hirudo officinalis* was included as outgroup. Clades corresponding to species were collapsed, with depth of the triangles representing intraspecific genetic divergence and height representing intraspecific haplotype diversity.

![](2_Figures/Decaëns et al Fig 4 Raref.pdf){ width=100% }
<span style="font-variant:small-caps;">Fig. 4.</span> -- Earthworm species diversity in the Mitaraka range: A) Incidence--based rarefaction and extrapolation curve showing species number accumulation as a function of the number of plots sampled within the study area; B) Abundance--based rarefaction and extrapolation curves showing species number accumulation as a function of the number of earthworm individuals sampled within four main types of habitats (Inselberg habitats include rocky savannahs, transition and hiltop forests); B) Abundance--based rarefaction and extrapolation curves showing species number accumulation as a function of the number of earthworm individuals sampled within four main types of microhabitats (Litter = litter accumulations on granitic rocks; Roots = root mat of terrestrial bromeliads; Soil = organomineral soil and sediments; Trunks = decaying trunks and associated termite mounds); D) Box--plots of community-scale species richness for the four main type of habitats (as in B). In A--C: solid curves = rarefaction curves; dotted curves = extrapolation curves. In D: diamond symbol = mean; colored dots = outliers; grey dots = individual values.

```{r, include=FALSE}
synt<-read.csv2("1_Datasets/Distancias_martio.csv", h=T)
Table<-flextable(synt) 
Table <- italic(Table, j = 1)
Table <- fontsize(Table, size = 8, part = "all")
Table <- autofit(Table, add_w = -0.05)
```
\newpage
<span style="font-variant:small-caps;">Table 2.</span> -- Uncorrected average pairwise genetic distances within (diagonal) and between species (under diagonal) of *Martiodrilus* included in this work.
`r knit_print(Table)`


```{r, include=FALSE}
synt<-read.csv2("1_Datasets/Distancias_acantho.csv", h=T)
Table<-flextable(synt) 
Table <- italic(Table, j = 1)
Table <- fontsize(Table, size = 8, part = "all")
Table <- autofit(Table, add_w = -0.05)
```
<span style="font-variant:small-caps;">Table 3.</span> -- Uncorrected average pairwise genetic distances within (diagonal) and between species (under diagonal) of Acanthodrilidae included in this work.
`r knit_print(Table)`

\newpage
# TAXONOMY

::: {.center data-latex=""}
Phylum ANNELIDA Lamarck, 1802

Class CLITELLATA Michaelsen, 1919

Sub-class OLIGOCHAETA Grube, 1850

Order CRASSICLITELLATA Jamieson, 1988


Family RHINODRILIDAE Benham, 1890

\vspace{12pt}
**Genus *Andiorrhinus* Cognetti de Martiis, 1908**
:::
*Andiorrhinus* -- Cognetti de Martiis, 1908. – Michaelsen 1918. – Cordero 1945. – Righi 1971. – Righi 1986. – Righi 1993. – Zicsi 1995. – Feijoo *et al.* 2017.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Andiorrhinus salvadorii* Cognetti de Martiis, 1908

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae arranged in four pairs of regular longitudinal lines. Intra-clitellar male pores. Gizzard in VI. Three pairs of calciferous glands in VII--IX, with lamellar structure.  [@cognettidemartiisLombrichiDiCosta1908; @righiVenezuelanEarthwormsConsideration1993; @feijoNewSpeciesAndiorrhinus2017]. Testes in X--XI within testis sacs. Two pairs of seminal vesicles in XI--XII. Spermathecae in pre-testicular segments, may be missing. Ovaries in XIII.

Five sub-genera are recognized, which differ from each other according to the number and position of intestinal hearts:

- *Andiorrhinus (Turedrilus)* Righi, 1993 has the last pair of hearts in XI;

- *Andiorrhinus (Andiorrhinus)* Cognetti de Martiis, 1908 has two pairs of intestinal hearts, with the last pair in XII;

- *Andiorrhinus (Quibario)* Feijoo *et al.*, 2017 has three pairs of intestinal hearts, with the last pair in XII;

- *Andiorrhinus (Meridrilus)* Righi, 1993 has two pairs of intestinal hearts, with the last pair in XIII;

- *Andiorrhinus (Amazonidrilus)* Righi, 1993 has three pairs of intestinal hearts, with the last pair in XIII.

<span style="font-variant:small-caps;">Diversity and distribution.</span> -- *Andiorrhinus* comprises 47 valid species (not including the new species described here), which are distributed in Venezuela (22 species), Brazil (17 species), Bolivia (four species), Colombia (three species), Guyana (one species), Paraguay (one species) (http://taxo.drilobase.org/; @misirliogluEarthwormsClitellataMegadrilisubmitted; @brownChecklistMegadrileEarthwormsubmitted).

\vspace{12pt}
## ***Andiorrhinus (Turedrilus) pascalei* Bartz & Decaëns n. sp.**

## (Fig. 5)

```{r, include=FALSE}
c2<-c["Andiorrhinus pascalei",]
m2<-subset(m, Species == "Andiorrhinus pascalei")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"],"; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"],"; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • One `r ht[,"Life.Stage"]` specimen (with regeneration); `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; Coll. MNHN (BOLD Sample ID: `r ht[,"Sample.ID"]`).

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimen; `r x1` Coll. CEFE (BOLD Sample ID: `r pt[,"Sample.ID"]`).

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` adult specimen, `r by(om,om$Life.Stage,nrow)[2]` juvenile specimens; `r x` Coll. MNHN (BOLD Sample ID: `r om[,"Sample.ID"]`).

<span style="font-variant:small-caps;">Etymology.</span> -- ThisThis species is named in acknowledgment to Olivier Pascal, who organised the “Our Planet Reviewed” in the Mitaraka Massif.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 5A)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]`. Male pores `r c2[,"Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 5B,C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]` (Fig. 5B). Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]` (Fig. 5C).

<span style="font-variant:small-caps;">Ecology.</span> -- *A. pascalei* n. sp. was found in the organomineral soil at the base camp of the expedition in a plateau forest ecosystem (Fig. 5D,E).

<span style="font-variant:small-caps;">Distribution.</span> -- *A. pascalei* n. sp. is known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *A. pascalei* n. sp. is attached to the subgenus *Turedrilus* by the position of the last pair of hearts in XI. Within Turedrilus it is close to *Andiorrhinus (Turedrilus) royeroi* Drachenberg, 1991, *Andiorrhinus (Turedrilus) samuelensis* Righi, 1986, *Andiorrhinus (Turedrilus) caudatus* Righi, Ayres & Bittencourt, 1976, *Andiorrhinus (Turedrilus) baniwa* Righi & Nemeth, 1983, *Andiorrhinus (Turedrilus) brunneus* (Michaelsen, 1892) and *Andiorrhinus (Turedrilus) meansi* James, 2009, all of which have three pairs of spermathecae [@feijoNewSpeciesEarthworms2012]. It differs from these species, however, by the position of the spermathecae in VII--IX instead of VI--VIII, and by the unique position of its clitellum and tubercula pubertatis. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.


\newpage
![](2_Figures/Decaëns et al Fig 5 Andiorrhinus pascalei 084.pdf){ width=97% }

<span style="font-variant:small-caps;">Fig. 5.</span> -- *Andiorrhinus pascalei* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position on the dorsal side, and of the tubercula pubertalis on the ventral side); B) Detail of the calciferous glands after being extracted from the dissection; C) Detail of the dissection showing the spermathecae; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype specimen and © T. Decaëns; CL= clitellum; SP= spermathecae; TP= tubercula pubertalis; N= total number of specimens in the dataset.


\newpage
::: {.center data-latex=""}
\vspace{12pt}
**Genus *Martiodrilus* Michaelsen, 1936**
:::

*Martiodrilus* -- Michaelsen, 1936. – Righi 1995. – Zicsi & Csuzdi 1997. – Zicsi 2001. – Fragoso & Brown 2007. – Csuzdi & Pavlíček 2011.

*Thamnodrilus* Beddard, 1887 (part.) -- Michaelsen 1900. – Černosvitov 1935.

*Thamnodrilus (Thamnodrilus)* (part.) -- Michaelsen 1918. – Černosvitov 1934.– Fragoso & Brown 2007.

*Thamnodriloides* -- Gates, 1968.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Martiodrilus heterostichon* (Schmarda, 1861)

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae arranged in eight regular longitudinal lines. One pair of intra-clitellar male pores. Five to eight pairs of calciferous glands in segments VII--XI, XII, XIII, XIV of tubular structure or alveolar structure. Genital apparatus holandric and metagynous. Seminal vesicles generally short, limited to one segment. spermathecae in pre-testicular segments [@righiSobreFamiliaGlossoscolecidae1971]. Ovaries in XIII.

Four sub-genera are separated depending on the type of anterior septa, the type of nephridia and the number of spermathecae [@csuzdiNewLittleKnown2011]:

- *Martiodrilus (Martiodrilus)* Michaelsen, 1936 has strongly thickened septa after the gizzard and nephridia with simple nephrostome;

- *Martiodrilus (Cordilleroscoloex)* Zicsi & Csuzdi, 1997 has strongly thickened septa after the gizzard and nephridia with multiple nephrostomes;

- *Martiodrilus (Botaria)* Zicsi, 1998 has simple nephrostome, membranous septa after gizzard and two to three pairs of spermathecae;

- *Martiodrilus (Maipure)* (Righi, 1995) has simple nephrostome, membranous septa after gizzard and four to six pairs of spermathecae.

<span style="font-variant:small-caps;">Distribution.</span> -- *Martiodrilus* comprises 85 valid species (not including the new species described here) distributed in South America, in Ecuador (57 species), Colombia (22 species), French Guiana (seven species), Peru (six species), Brazil (three species), Surinam (three species), Panama (two species) and Guyana (one species) (http://taxo.drilobase.org/; @misirliogluEarthwormsClitellataMegadrilisubmitted; @brownChecklistMegadrileEarthwormsubmitted). 

\vspace{12pt}
## ***Martiodrilus (Botaria) flavus* Decaëns & Bartz n. sp.**

## (Fig. 2F,6)

```{r, include=FALSE}
c2<-c["Martiodrilus flavus",]
m2<-subset(m, Species == "Martiodrilus flavus")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"],"; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"],"; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • One `r ht[,"Life.Stage"]` specimen (with regeneration); `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: 3 specimens at CEFE, 2 at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens, `r by(om,om$Life.Stage,nrow)[2]` cocoons; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the remarkable yellow pigmentation of the body observed in most adult specimens.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 6A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 6A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 6B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 6C-F)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]` (Fig. 6E, F). Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]` (Fig. 6D).

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (B.) flavus* n. sp. was predominantly found in plateau forests (`r round(hab[1]*100/sum(hab), digits=2)` % of specimens) and other well drained habitats such as slope forests (`r hab[2]*100/sum(hab)` %) and transition forest at the edge of rocky savanna (`r hab[3]*100/sum(hab)` %) (Fig. 6G). It preferentially inhabits under large decaying trunks fallen at the soil surface (`r mic[1]*100/sum(mic)` %), but was also occasionally found within the soil (`r mic[2]*100/sum(mic)` %) (Fig. 6H). Cocoons are tan yellow spherical, 15--20 mm in diameter, with marked mucrons at the anterior and posterior ends.

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) flavus* n. sp. is known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- Of the *Martiodrilus* species found in French Guiana, *M. (B.) flavus* n. sp. is easily distinguished by its relatively large size and yellow to grey skin pigmentation. The presence of three pairs of spermathecae links it to *Botaria*, however it differs from all other known species in the sub-genus by their position in 5/6--7/8 instead of 6/7--8/9. It also differs in the position of the seminal vesicles in X--XI instead of XI--XII in other species of *Martiodrilus*. It is similar in the position of the clitellum to *Martiodrilus (Botaria) vassae* Zicsi & Czusdi, 1999 and *Martiodrilus (Botaria) gara* Righi, 1995, but differs in the position of the tubercula pubertatis in XXIV--XV instead of XXI--XXV in *M. (B.) vassae* or XIX--XXIII, 1/2XXIV in *M. (B.) gara*. *M. (B.) flavus* n. sp. is clearly separated from the other *Martiodrilus* species of the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 6.</span> (next page) -- *Martiodrilus (Botaria) flavus* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and semi-ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Detail on the spermathecae; E) Detail on the typhlosole when rolled up; F) Detail on the typhlosole when unrolled; G) Habitat preferences; H) Microhabitat preferences. All images from the Holotype specimen and © T. Decaëns; CL= clitellum; DV= dorsal vessel; GZ= gizard; IN= intestine; SP= spermathecae; SV= seminal vesicles; TF= typhlosole; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 6 Martiodrilus flavus 064.pdf){ width=97% }
<span style="font-variant:small-caps;">Fig. 6.</span>

\newpage
## ***Martiodrilus (Botaria) lavellei* Decaëns & Bartz n. sp.**

## (Fig. 7)

```{r, include=FALSE}
c2<-c["Martiodrilus lavellei",]
m2<-subset(m, Species == "Martiodrilus lavellei")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in honour of Patrick Lavelle, in acknowledgement to his contribution for the study of tropical earthworm ecology.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 7A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves passing through segment I in line of nephridiopores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 7A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 7B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 7C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- This species is only known from the Holotype specimen, has been collected in the organo-mineral soil of a plateau forests (Fig. 7D,E).

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) lavellei* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (B.) lavellei* belongs to the sub-genus *Botaria* because of the presence of three pairs of spermathecae. It differs from the majority of species in the sub-genus its clitellum beginning in XVI, a position which it shares only with the two Guyanese species *Martiodrilus (Botaria) dewynteri* Csuzdi & Pavlicek (2011) and *Martiodrilus (Botaria) helleri* (Michaelsen, 1918). However, the clitellum of *M. (B.) lavellei* is less extended posteriously than in these two species, ending in XXV instead of XXVI. It is also significantly smaller than *M. (B.) helleri*, and appears to have significantly smaller spermathecae than *M. (B.) dewynteri* as far as the illustration in @csuzdiNewLittleKnown2011 allows us to judge. *M. (B.) lavellei* n. sp. is clearly separated from other species of *Martiodrilus* from the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 7 Martiodrilus lavellei 080.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 7.</span> -- *Martiodrilus lavellei* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; GZ= gizard; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
## ***Martiodrilus (Botaria) motoloi* Bartz & Decaëns n. sp.**

## (Fig. 8)

```{r, include=FALSE}
c2<-c["Martiodrilus motoloi",]
m2<-subset(m, Species == "Martiodrilus motoloi")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```


<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: one specimen at CEFE, one specimen at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` fragment, `r by(om,om$Life.Stage,nrow)[2]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named after the Wayana Amerindian people, in whose language motoloi means earthworm.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 8A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves going anterior to posterior through segment I in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 8A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 8B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 8C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (B.) motoloi* n. sp. has been found in lowland forests  (Fig. 8D), in decaying trunks (`r round(mic[1]*100/sum(mic), digits=2)` % of specimens), organomineral soil (`r round(mic[2]*100/sum(mic), digits=2)` % of specimens), and in litter accumulation on granite rocks in decaying trunks (`r round(mic[3]*100/sum(mic), digits=2)` % of specimens) (Fig. 8G).

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) motoloi* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (B.) motoloi* n. sp belongs to the sub-genus *Botaria* because of the presence of three pairs of spermathecae. It resembles other species of this sub-genus which share a clitellum in XV—XXVI, including *Martiodrilus (Botaria) benhami* (Cognetti, 1904), *Martiodrilus (Botaria) feijooi* Zicsi, 1998, *M. (B.) gara*, *Martiodrilus (Botaria) gravis* (Cognetti, 1904), *Martiodrilus (Botaria) pano* Righi, 1992, *M. (B.) vassae* and *Martiodrilus (Botaria) wayana* n. sp.. It differs from all these species but *M. (B.) gravis* and *M. (B.) benhami* by the position of its tubercula pubertatis in 1/3 XX--1/2 XXV. It can be distinguished from both *M. (B.) gravis* and *M. (B.) benhami* by its smaller size (42 to 56 mm instead of more 90 mm and 170 mm, respectively), and by its darker pigmentation. *M. (B.) motoloi* n. sp. is clearly separated from other species of *Martiodrilus* from the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 8 Martiodrilus motoloi 088.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 8.</span> -- *Martiodrilus (Botaria) motoloi* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; DV= dorsal vessel; GZ= gizard; IN= intestine; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
## ***Martiodrilus (Botaria) pavliceki* Bartz & Decaëns n. sp.**

## (Fig. 9)

```{r, include=FALSE}
c2<-c["Martiodrilus pavliceki",]
m2<-subset(m, Species == "Martiodrilus pavliceki")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in honour of Tomas Pavlicek, in acknowledgement to his contribution for the study of earthworms in French Guiana and in other regions of the World.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 9A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves going anterior to posterior through segment I in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 9A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 9B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 9C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (B.) pavliceki* n. sp. has been found in slope forests (`r round(hab[2]*100/sum(hab), digits=2)` % of specimens) and in plateau forests (`r round(hab[1]*100/sum(hab), digits=2)` %) (Fig. 9D). All specimens were collected in the organo-mineral soil (Fig. 9E).

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) pavliceki* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M.(B.) pavliceki* resembles species of sub-genus *Botaria* which share a clitellum in XV--XXV. It differs from all these species, and from *Martiodrilus (Botaria) tchoukouchipan* by the unique position of its tubercula pubertatis in XIX--1/n XXV. It is also noticeably larger than *M. (B.) tchoukouchipan* (95 mm vs. 55--75 mm in the latter), and has flattened sac-like spermathecae whereas they are elongated in *M. (B.) tchoukouchipan*. *M. (B.) pavliceki * n. sp. is clearly separated from other species of *Martiodrilus* from the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 9 Martiodrilus pavliceki 076.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 9.</span> -- *Martiodrilus (Botaria) pavliceki* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; GZ= gizard; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
## ***Martiodrilus (Botaria) tchoukouchipan* Decaëns & Bartz n. sp.**

## (Fig. 2E,10)

```{r, include=FALSE}
c2<-c["Martiodrilus tchoukouchipan",]
m2<-subset(m, Species == "Martiodrilus tchoukouchipan")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: 4 specimens at CEFE, 6 specimens at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` adult specimens, `r by(om,om$Life.Stage,nrow)[4]` juvenile specimen, `r by(om,om$Life.Stage,nrow)[3]` fragment, `r by(om,om$Life.Stage,nrow)[2]` cocoons; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the Tchoukou Chipan inselberg nearby which the holotype specimen was collected.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 10A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. OneOne pair of longitudinal grooves going anterior to posterior through segment I in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 10A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 10B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 10C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (B.) tchoukouchipan* n. sp. has been found in a wide range of well-drained habitats, with a marked preference for rocky savannahs of the inselbergs (`r round(hab[2]*100/sum(hab), digits=2)` % of specimens), plateau forests (`r round(hab[4]*100/sum(hab), digits=2)` %) and inselberg transition forests (`r round(hab[3]*100/sum(hab), digits=2)` %) (Fig. 10D). Most specimens were collected under terrestrial bromeliads covering rocky savannas (`r round(mic[3]*100/sum(mic), digits=2)` % of specimens), in the organo-mineral soil (`r round(mic[4]*100/sum(mic), digits=2)` %) and a lower proportion in decaying trunks (`r round(mic[1]*100/sum(mic), digits=2)` %) and in litter accumulating on large granite rocks (`r round(mic[2]*100/sum(mic), digits=2)` %)  (Fig. 10E). Cocoons are tan yellow spherical, 4.5--5.5 mm in diameter, with two mucrons at the anterior and posterior ends.

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) tchoukouchipan* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (B.) tchoukouchipan* n. sp. belongs to the sub-genus *Botaria* because of the presence of 3 pairs of spermathecae. It resembles a group of species sharing a clitellum positioned on XV--XXV, including *Martiodrilus (Botaria) bicolor* (Michaelsen, 1913), *Martiodrilus (Botaria) euzonus* (Cognetti, 1904), *M. (B.) gara*, *M. (B.) gravis* (Cognetti, 1904), *Martiodrilus (Botaria) minoriformis* Zicsi, 1998, *Martiodrilus (Botaria) poncei* Zicsi, 1998 and *Martiodrilus (Botaria) tutus* (Cognetti, 1904). However, it differs from these species in the position of the tubercula pubertatis in 1/4 XX--1/3 XXV which is only compatible with the description of *M. (B.) gravis* and *M. (B.) tutus*. It is finally distinguished from the latter two species by its significantly smaller size (55--75 mm, whereas *M. (B.) gravis* and *M. (B.) tutus* are larger than 90 mm and 100 mm, respectively). *M. (B.) tchoukouchipan* n. sp. is clearly separated from the other species of *Martiodrilus* from the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 10 Martiodrilus tchoukouchipan 071.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 10.</span> -- *Martiodrilus (Botaria) tchoukouchipan* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CG= calciferous glands; CL= clitellum; GZ= gizard; SP= spermathecae; SV= seminal vesicles; TF= typhlosole; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
## ***Martiodrilus (Botaria) wayana* Bartz & Decaëns n. sp.**

## (Fig. 11)

```{r, include=FALSE}
c2<-c["Martiodrilus wayana",]
m2<-subset(m, Species == "Martiodrilus wayana")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; desposited as follow: three specimens at CEFE, three specimens at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named after the Wayanas Amerindian people who inhabited this region of French Guiana and have left multiple petroglyphs and stone assemblages on the Borne 1 inselberg [@fleuryArcheologieParticipativeAvec2021].

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 11A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves going anterior to posterior through segment I in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 11A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 11B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 11C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (B.) wayana* n. sp. has been found essentially in inselberg rocky savannahs (`r round(hab[2]*100/sum(hab), digits=2)` % of specimens) (Fig. 11D), where it inhabits the root mat of terrestrial bromeliad formations (Fig. 11G).

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (B.) wayana* n. sp. is only known from the Borne 1 inselberg, at the frontier between French Guiana and Brazil, in the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (B.) wayana* n. sp belongs to the sub-genus *Botaria* because of the presence of three pairs of spermathecae. It resembles other species of this sub-genus which share a clitellum in XV—XXVI, including *M. (B.) benhami*, *Martiodrilus (Botaria) feijooi* Zicsi, 1998, *M. (B.) gara*, *M. (B.) gravis*, *Martiodrilus (Botaria) pano* Righi, 1992, *M. (B.) vassae* and *M. (B.) motoloi* n. sp.. It differs from all these species but *M. (B.) pano*, by the position of its tubercula pubertatis in XX--1/2 XXVI, and it further differs from *M. (B.) pano* by its smaller size (55 to 74 mm instead of 85 mm) and by the more anterior transition between oesofagus and intestine. *M. (B.) wayana* n. sp. is clearly separated from other species of *Martiodrilus* from the Mitaraka Massif by its COI barcode (Table 2). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 11 Martiodrilus wayana 083.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 11.</span> -- *Martiodrilus (Botaria) wayana* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; DV= dorsal vessel; GZ= gizard; IN= intestine; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
## ***Martiodrilus (Maipure) luminescens* Decaëns & Bartz n. sp.**

## (Fig. 12)

```{r, include=FALSE}
c2<-c["Martiodrilus luminescens",]
m2<-subset(m, Species == "Martiodrilus luminescens")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: two specimens at CEFE, one at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[2]` juvenile specimens, `r by(om,om$Life.Stage,nrow)[1]` cocoon; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- The name of the species refers to the bioluminescent character observed on one of the type specimens during field sampling.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 12A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 12A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 12B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 12C-E)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]` (Fig. 12E). Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]` (Fig. 12E). Spermathecae: `r c2[,"Spermatheca"]` (Fig. 12D).

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (M.) luminescens* n. sp. has been found mostly in slope forests (`r round(hab[3]*100/sum(hab), digits=2)` % of specimens) and in inselberg rocky savannahs (`r round(hab[1]*100/sum(hab), digits=2)` %) and transition forests (`r round(hab[2]*100/sum(hab), digits=2)` (Fig. 12F). Most specimens were collected in decaying trunks (`r round(mic[1]*100/sum(mic), digits=2)` %), in litter accumulations on granite rocks (`r round(mic[2]*100/sum(mic), digits=2)` %) and in the root mat of terrestrial bromeliads (`r round(mic[3]*100/sum(mic), digits=2)` %) (Fig. 12G). Cocoons are tan yellow spherical, 10 mm in diameter, with two mucrons at the anterior and posterior ends.

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (M.) luminescens* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (M.) luminescens* n. sp. belongs to the sub-genus *Maipure* due to the presence of four pairs of spermathecae. It is related to other species sharing a clitellum in XV--XXVI, such as *Martiodrilus (Maipure) micrurus* Cognetti, 1904, *Martiodrilus (Maipure) potarensis* Rosa, 1895, *Martiodrilus (Maipure) ophioides* (Cognetti, 1904) and *Martiodrilus (Maipure) rigeophilus* Cognetti, 1904. It differs from the first two species in the position of the tubercula pubertalis, and also from *M. (M.) potarensis* in its significantly smaller body size (109--170mm instead of 380mm). *M. (M.) luminescens* is also significantly larger than *M. (M.) rigeophilus* and *M. (M.) ophioides* which have body sizes ranging from 70 to 80mm and from 100 to 120mm, respectively. Furtheremore, both of these two last species are known from high-altitude montane ecosystems (above 4,000 and 2,500m asl, respectively) which are quite different from the type habitat in which *M. (M.) luminescens* was found. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 12.</span> (next page) -- *Martiodrilus (Maipure) luminescens* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Detail of the anterior part of the dissection showing the spermathecae; E) Detail of the dissection shwoing the seminal vessicle and the calciferous glands;  F) Habitat preferences; G) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; DV= dorsal vessel; GZ= gizard; HT= hearts; IN= intestine; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 12 Martiodrilus luminescens 082.pdf){ width=97% }
<span style="font-variant:small-caps;">Fig. 12.</span>

\newpage
## ***Martiodrilus (Maipure) mitaraka* Decaëns & Bartz n. sp.**

## (Fig. 13)

```{r, include=FALSE}
c2<-c["Martiodrilus mitaraka",]
m2<-subset(m, Species == "Martiodrilus mitaraka")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ] 
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: four specimens at CEFE, ten at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[2]` juvenile specimen, `r by(om,om$Life.Stage,nrow)[1]` cocoons; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the Mitaraka Massif where the type material has been collected.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 13A-B, D)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves going anterior to posterior through segment I in line of nephridial pores (Fig. 13D). Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 13A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 13B). Male pores `r c2[,"Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 13C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. (M.) mitaraka* n. sp. has been found mostly in plateau forests (`r round((hab[2]+hab[4])*100/sum(hab), digits=2)` % of specimens) and in inselberg transition forests (`r round(hab[1]*100/sum(hab), digits=2)` %) (Fig. 13E). Most specimens were collected in the organo-mineral soil (`r round(mic[4]*100/sum(mic), digits=2)` %) and to a lower extent in litter accumulation on granit rocks (`r round(mic[3]*100/sum(mic), digits=2)` %) and in decaying trunks (`r round((mic[1]+mic[2])*100/sum(mic), digits=2)` %) (Fig. 13F). Cocoons are tan yellow spherical, 7--10 mm in diameter, with two mucrons at the anterior and posterior ends.

<span style="font-variant:small-caps;">Distribution.</span> -- *M. (M.) mitaraka* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (M.) mitaraka* n. sp. belongs to the sub-genus *Maipure* due to the presence of four pairs of spermathecae. It is similar to the species of this sub-genus whose clitellum is in XV--XXV: *Martiodrilus (Maipure) ecuadoriensis* (Benham 1892), *Martiodrilus (Maipure) palmirus* Zicsi & Feijoo, 1994, and *M. (M.) ophioides*. It also shares with the latter the position of the tubercula pubertalis in XX--XXVI. It differs from these three species, however, in the position of spermathecal pores in 4/5--7/8, whereas in the latter they are one segment more posterior. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 13.</span> (next page) -- *Martiodrilus (Maipure) mitaraka* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Detail of anterior segments (external lateral view); E) Habitat preferences; F) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; GZ= gizard; SP= spermathecae; SV= seminal vesicles; TF= tiphlosole; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 13 Martiodrilus mitaraka 081.pdf){ width=97% }
<span style="font-variant:small-caps;">Fig. 13.</span>

\newpage
## ***Martiodrilus (Maipure) terricophilous* Bartz & Decaëns n. sp.**

## (Fig. 14)

```{r, include=FALSE}
c2<-c["Martiodrilus terricophilous",]
m2<-subset(m, Species == "Martiodrilus terricophilous")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ] 
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow) 
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```


<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: two specimens at CEFE, one at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the fact that unlike most *Martiodrilus* species in the Mitaraka region, it seems to inhabit the organomineral soil rather than epigeic microhabitats.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 14A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 14A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 14B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 14C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *M. terricophilous* n. sp. has been found essentially in the organomineral soil of plateau forest ecosystems (Fig. 14D,E).

<span style="font-variant:small-caps;">Distribution.</span> -- *M. terricophilous* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *M. (M.) terricophilous* n. sp. belongs to the subgenus *Maipure* due to  the presence of four pairs of spermathecae. It is similar to *M. (M.) ophioides* which may also have a clitellum in XIV--XXV and has tubercula pubertalis in XX--XXVI. *M. (M.) terricophilous* is much larger in body size than *M. (M.) ophioides* (170--225mm instead of 100--120mm), from which it also differs in the position of the spermathecal pores in 4/5--7/8 instead of 5/6--8/9. Furtheremore, *M. (M.) terricophilous* occurs in a radically different habitat from the Andean forests from which *M. (M.) ophioides* is known. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 14.</span> (next page) -- *Martiodrilus terricophilous* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; DV= dorsal vessel; GZ= gizard; IN= intestine; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 14 Martiodrilus terricophilous 085.pdf){ width=100% }
<span style="font-variant:small-caps;">Fig. 14.</span>

\newpage
::: {.center data-latex=""}
**Genus *Urobenus* Benham, 1887**
:::
*Urobenus* Benham, 1887; Benham 1890; Beddard 1895; Righi 1985.

*Anteus* Perrier, 1872 (part.) -- Rosa 1896.

*Rhinodrilus* Perrier, 1872 (part.) -- Michaelsen 1900, 1918; Stephenson 1930; Cordero 1945; Righi 1971; Jamieson 1971.

*Rhinodrilus (Rhinodrilus)* (part.) -- Cognetti 1906.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Urobenus brasiliensis* Benham, 1887

<span style="font-variant:small-caps;">Diagnosis.</span> -- Steae arranged in eight regular longitudinal lines. One pair of intra-clitellar male pores. Gizzard in VI. Three pairs of calciferous glands in VII--IX, those of VII and VIII wholly or mostly of tubular-paniculated structure, those of IX saccular, with thin wall and wide cavity endowed with irregular and sparse folds. Hearts in VII--XI. Genital apparatus holandric and metagynous. Seminal vesicles short. spermathecae in pre-testicular segments. Ovaries in XIII [@righiSobreRhinodrilusUrobenus1985].

<span style="font-variant:small-caps;">Distribution.</span> -- *Urobenus* comprises six valid species (not including the new species described here) distributed in Brazil, with *Urobenus brasiliensis* (Benham, 1886) also reported from Paraguay (http://taxo.drilobase.org/; @misirliogluEarthwormsClitellataMegadrilisubmitted; @brownChecklistMegadrileEarthwormsubmitted). 

\vspace{12pt}
## ***Urobenus guianomeridionalis* Bartz & Decaëns n. sp.**

## (Fig. 15)

```{r, include=FALSE}
c2<-c["Urobenus guianomeridionalis",]
m2<-subset(m, Species == "Urobenus guianomeridionalis")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: three specimen at CEFE, three specimen at MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the type locality situated in the south of French Guiana.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 15A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. One pair of longitudinal grooves going anterior to posterior through segment I--II in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 15A). Genital markings `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 15B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 15C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *U. guianomeridionalis* n. sp. has been found in plateau forests (`r round(hab[3]*100/sum(hab), digits=2)` % of specimens), in inselberg hiltop and transition forests (`r round((hab[1]+hab[2])*100/sum(hab), digits=2)` % of specimens), and in slope forests (`r round(hab[4]*100/sum(hab), digits=2)` % of specimens) (Fig. 15D). It is a specialist of epigeic microhabitats such as decaying trunks and litter accumulations on granite rocks (Fig. 15G).

<span style="font-variant:small-caps;">Distribution.</span> -- *U. guianomeridionalis* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *Urobenus guianomeridionalis* n. sp. is close to *Urobenus brevis* (Omodeo, 1955) and *Urobenus brasiliensis* Benham, 1887 in the position of the clitellum in XV--XXIV [@benhamStudiesEarthwormsII1887; @omodeoOligochetiTerricoliVenezuela1955]. It also shares with *U. brevis* the paniculate tubular structure of the calcified glands in VII--VIII, which differentiates both species from all others in the genus, and the position of the male pore in 19/20. The two species differ mainly in the much smaller size of *U. guianomeridionalis* n. sp. (35 to 41 mm in length and 1.2 to 2.8 mm in anterior diameter, compared to 90 mm and 8 mm in *U. brevis*, respectively), the setal ratio (3:1:4:1:13, compared to 9:1:10:1:38 in *U. brevis*) and spermathecae lacking seminal chambers in *U. guianomeridionalis* n. sp. [@righiSobreRhinodrilusUrobenus1985]. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 15.</span> (next page) -- *Urobenus guianomeridionalis* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the position of the tubercula pubertalis); C) Dissection showing the internal anatomy of the anterior body; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; GZ= gizard; SP= spermathecae; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 15 Urobenus guianomeridionalis 052.pdf){ width=100% }
<span style="font-variant:small-caps;">Fig. 15.</span>

\newpage
::: {.center data-latex=""}
**Genus *Atatina* Righi, 1971**
:::

*Atatina* Righi, 1971.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Atatina puba* Righi, 1971

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae arranged in eight regular longitudinal lines. One pair of intra-clitellar male pores. Gizzard in VI. Four to five pairs of calciferous glands in segments IX--XIV, of trabecular structure, sacculiform and opening widely into the oesophagus. Genital apparatus metandric and metagynous. Seminal vesicles long, extended posteriously across many segments. spermathecae in pre-testicular segments. Ovaries in XIII [@righiSobreFamiliaGlossoscolecidae1971; @righiOligochaetaAnnelidaInstituto].

<span style="font-variant:small-caps;">Distribution.</span> -- *Atatina* comprises two species from Brazil (not including the species described here) (http://taxo.drilobase.org/; @misirliogluEarthwormsClitellataMegadrilisubmitted; @brownChecklistMegadrileEarthwormsubmitted).

\vspace{12pt}
## ***Atatina albida* Bartz & Decaëns n. sp.**

## (Fig. 16)

```{r, include=FALSE}
c2<-c["Atatina albida",]
m2<-subset(m, Species == "Atatina albida")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: two specimens at CEFE, eight specimens at MNHN. 

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- The name of the species refers to the absence of pigmentation which characterises it.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 16A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. OneOne pair of longitudinal grooves going anterior to posterior through segment I--II in line of nephridial pores. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd:dd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`:`r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]` (Fig. 16A). Genital markings in `r c2[,"Marks"]`. Tubercula pubertatis `r c2[,"Puberculum"]` (Fig. 16B). Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 16C)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`, with an average size (width x length) of `r c2[,"Gizzard_size"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *A. albida* n. sp. has been found in a range of ecosystems including lowland, slope, planteau, transition and Heliconia forests (Fig. 16D). It has been observed in the organimineral soil (`r round(mic[2]*100/sum(mic), digits=2)` % of specimens) and in decaying truncs (`r round(mic[1]*100/sum(mic), digits=2)` % of specimens) (Fig. 16G).

<span style="font-variant:small-caps;">Distribution.</span> -- *A. albida* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *A. albida* differs from *Atatina gatesi* (Righi & Ayres, 1978) and *Atatina puba* (Righi, 1971), the only other known species in the genus, in the position of the clitellum and tubercula pubertatis, as well as in the presence of two pairs of spemathecae instead of one in these two relatives [@righiOligochaetaAnnelidaInstituto; @righiSobreFamiliaGlossoscolecidae1971]. It shares with *A. gatesi* the presence of four pairs of calciferous glands, but their position is different (at IX--XII in *A. albida* n. sp. vs. XI--XIV in *A. gatesi*). It also shares with *A. puba* the presence of a single pair of seminal vesicles in XI. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

<span style="font-variant:small-caps;">Fig. 16.</span> (next page) -- *A. albida* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum and tubercula pubertalis positions); B) Dissection showing the internal anatomy of the anterior body; C) Detailed view of the dissection showing the calciferous glands; D) Habitat preferences; E) Microhabitat preferences. All images from the Holotype, except C) (Paratype EW-MI15-0121); all images © T. Decaëns; CL= clitellum; GZ= gizard; IN= intestine; SV= seminal vesicles; TP= tubercula pubertalis; N= total number of specimens in the dataset.

\newpage
![](2_Figures/Decaëns et al Fig 16 Atatina albida 074.pdf){ width=90% }

<span style="font-variant:small-caps;">Fig. 16.</span>

\newpage
```{r, include=FALSE}
synt<-read.csv2("1_Datasets/Synthesis.csv", h=T)
Table<-flextable(synt) 
Table <- italic(Table, j = 1)
Table <- fontsize(Table, size = 8, part = "all")
```
<span style="font-variant:small-caps;">Table 4.</span> -- Main morpho-anatomical characteristics of the new Rhinodrilidae species from the Mitaraka range.
`r knit_print(Table)`

\newpage
::: {.center data-latex=""}
Family ACANTHODRILIDAE Claus, 1880

Subfamily BENHAMIINAE Michaelsen, 1897

\vspace{12pt}
**Genus *Neogaster* Cernosvitov, 1934**
:::
*Neogaster* Cernosvitov, 1934; Righi & Caballero 1970; Righi 1988.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Neogaster americanus* Cernosvitov, 1934

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae arranged in eight regular longitudinal lines. Clitellum extending from segments XIII to XX. Weak gizzard in VI. Excretory system of meronephridial type. Two pairs of calciferous glands in XIV—XV, of lamellar type and opening in the oesophagus throught a common duct. Genital apparatus acanthodrilic. Spermathecae in pre-testicular segments, paired and without diverticula [@cernosvitovOligochetesGuyaneFrancaise1934; @csuzdiMonographPaleotropicalBenhamiinae2010].

<span style="font-variant:small-caps;">Distribution.</span> -- *Neogaster* comprises four Neotropical species (not including the new species described here), all known from the state of Amapá in Brazil [@cernosvitovOligochetesGuyaneFrancaise1934; @righiUmaColecaoOligochaeta; @righiDuasNovasEspecies1970; @brownChecklistMegadrileEarthwormsubmitted)].

\vspace{12pt}
## ***Neogaster csuzdii* Bartz & Decaëns n. sp.**

## (Fig. 17)

```{r, include=FALSE}
c2<-c["Neogaster csuzdii",]
m2<-subset(m, Species == "Neogaster csuzdii")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: one specimen in CEFE, one specimen in MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in homage to Csaba Csuzdi, in recognition for his involvement in the study of the Acanthodrilidae family, and of his contribution to the description of new earthworm species for French Guiana.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 17A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.*

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: two pairs of `r c2[,"Nephridia"]` per segment. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- The three known specimens of *N. csuzdii* n. sp. have been found in a lowland, a slope and a transition forests (Fig. 17C), two in decaying trunks and one in the litter accumulating on a granite rock (Fig. 17D).

<span style="font-variant:small-caps;">Distribution.</span> -- *N. csuzdii* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *N. csuzdii* n. sp. is closely related to *Neogaster tumuchumac* n. sp. with which it shares small body size, the absence of a gizzard and the entirely membranous septa in the anterior part of the body (Table 5). It is distinguished from all other *Neogaster* by the presence of a single pair of spermathecae in IX, whereas the other species have two (in *N. tumuchumac* n. sp.), three (in *Neogaster angeloi* Righi, 1988 and *Neogaster americanus* Cernosvitov, 1934), four (in *Neogaster gavrilovi* Righi & Caballero, 1970) or five (in *Neogaster aidae* Righi, 1975). In *N. csuzdii*, we observed only one pair of seminal vesicles in XII, whereas two pairs are usually present in other species (but see remarks on *N. tumuchumac*).  *N. csuzdii* n. sp. is smaller than most of the other species (20-30 mm compared to over 40 mm in *N. americanus*, *N. aidae* and *N. gavrilovi*), and differs from the equally small *N. angeloi* by the presence of skin pigmentation, which the latter lacks [@cernosvitovOligochetesGuyaneFrancaise1934; @righiDuasNovasEspecies1970; @righiOligochaetaBrazilianAmazonia1975; @righiUmaColecaoOligochaeta]. *N. csuzdii* n. sp. is clearly separated from *N. tumuchumac* n. sp. from its COI barcode (Table 3). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 17 Neogaster csuzdii 009.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 17.</span> -- *Neogaster csuzdii* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the clitellum male area); C) Habitat preferences; D) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; MA= male area; N= total number of specimens in the dataset.

\newpage
## ***Neogaster tumuchumac* Decaëns & Bartz n. sp.**

## (Fig. 18)

```{r, include=FALSE}
c2<-c["Neogaster tumuchumac",]
m2<-subset(m, Species == "Neogaster tumuchumac")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: two specimens in CEFE, two specimens in MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in reference to the Tumuc Humac mountains to which the Mitaraka Massif belongs to.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 18A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.*

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: `r c2[,"Nephridia"]`. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- *N. tumuchumac* n. sp. has been found in decaying trunks in both slope (`r round(hab[2]*100/sum(hab), digits=2)` % of specimens) and plateau forests (`r round(hab[1]*100/sum(hab), digits=2)` % of specimens) (Fig. 18C,D).

<span style="font-variant:small-caps;">Distribution.</span> -- *N. tumuchumac* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- As previously stated, *N. tumuchumac* n. sp. is closely related to *N. csuzdii* n. sp. due to the absence of  gizzard and the membranous anterior septa (Table 5). It is the smallest known species within the genus, with body size not exceeding 17 mm in the studied material. It differs from other *Neogaster* by the presence of two pairs of spermathecae in VIII--IX, whereas other species have one (in *N. csuzdii* n. sp.), three (in *N. angeloi* and *N. americanus*), four (in *N. gavrilovi*) or five (in *N. aidae*) [@cernosvitovOligochetesGuyaneFrancaise1934; @righiDuasNovasEspecies1970; @righiOligochaetaBrazilianAmazonia1975; @righiUmaColecaoOligochaeta]. *N. tumuchumac* n. sp. is clearly separated from *N. csuzdii* n. sp. from its COI barcode (Table 3). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 18 Neogaster tumuchumac 077.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 18.</span> -- *Neogaster tumuchumac* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the clitellum male area); C) Habitat preferences; D) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; MA= male area; N= total number of specimens in the dataset.

\newpage
::: {.center data-latex=""}
**Genus *Omodeoscolex* Csuzdi, 1993**
:::
*Omodeoscolex* Csuzdi, 1993; Csuzdi 2010.

\vspace{12pt}
<span style="font-variant:small-caps;">Type species.</span> -- *Omodeoscolex divergens*  (Cognetti, 1905)

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae arranged in eight regular longitudinal lines. Clitellum extending from segments XIII to XX. Gizzard absent. Excretory system of holonephridial type. Two pairs of calciferous glands in XIV--XV, of lamellar type and opening in the oesophagus separately. Spermathecae in pre-testicular segments, paired and without diverticula [@csuzdiUberTaxonomischenProbleme1993; @csuzdiMonographPaleotropicalBenhamiinae2010].

<span style="font-variant:small-caps;">Distribution.</span> -- *Omodeoscolex* comprises two Neotropical species (not including the new species described here): *Omodeoscolex divergens* (Cognetti de Martiis, 1905) is reported from	Brazil, Ecuador, French Guiana and Panama (http://taxo.drilobase.org/), while *Omodeoscolex tocaya* (Righi, Ayres & Bittencourt, 1978) is known from Brasil (Amazonas) [@righiOligochaetaAnnelidaInstituto].

\vspace{12pt}
## ***Omodeoscolex touroulti* Decaëns & Bartz n. sp.**

## (Fig. 19)

```{r, include=FALSE}
c2<-c["Omodeoscolex touroulti",]
m2<-subset(m, Species == "Omodeoscolex touroulti")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: one specimen in CEFE, one specimen in MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- This species is named in acknowledgment to Julien Touroult, co-organisator of the "Our Planet Reviewed" in the Mitaraka Massif.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 19A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.*

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- the three specimens of *Omodeoscolex touroulti* n. sp. were found in decaying trunks and termite nest in a lowland, a slope and a plateau forest (Fig. 19C,D).

<span style="font-variant:small-caps;">Distribution.</span> -- *O. touroulti* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *O. touroulti* n. sp. is different from the two previously described species of *Omodeoscolex* by the presence of unpaired instead of paired spermathecae. Despite this strong difference, we provisionally assign the species to the genus *Omodeoscolex* because of lack of a gizzard and of the holonepridial type of its excretory system. *O. touroulti* n. sp. is close to  *O. abamensis* n. sp. in the presence of three spermathecae, instead of four in *O. tetraspermatheca* n. sp.. It however differs from *O. abamaensis* n. sp. by the position of the spermathecae in VI–VIII instead of VII–IX (Table 5). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021, and it is clearly separated from other *Omodeoscolex* species of Mitaraka by mean pairwise ditances above 16% (Table 3).

\newpage
![](2_Figures/Decaëns et al Fig 19 Omodeoscolex touroulti 012.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 19.</span> -- *Omodeoscolex touroulti* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the clitellum male area); C) Habitat preferences; D) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; MA= male area; N= total number of specimens in the dataset.

\newpage
## ***Omodeoscolex abamaensis* Bartz & Decaëns n. sp.**

## (Fig. 20)

```{r, include=FALSE}
c2<-c["Omodeoscolex abamaensis",]
m2<-subset(m, Species == "Omodeoscolex abamaensis")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited as follow: one specimen in CEFE, one specimen in MNHN.

**Other material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` fragment, `r by(om,om$Life.Stage,nrow)[2]` juvenile specimen; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Etymology.</span> -- The name of the species refers to the Abama River along which the expedition's base camp was located. 

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 20A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores: `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.*

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- the three specimens of *Omodeoscolex abamaensis* n. sp. were found in decaying trunks and termite nest in a lowland, a slope and a plateau forest (Fig. 20C,D).

<span style="font-variant:small-caps;">Distribution.</span> -- *O. abamaensis* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *O. abamaensis* is close to *O. touroulti* n. sp. and *O. tetraspermatheca* n. sp. in presence of unpaired spermathecae (Table 5). It however  differs from these closely relatives by its DNA barcode (mean pairwise divergence > 16% with these species; Table 3), and by the number and/or position of spermathecae. This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 20 Omodeoscolex abamaensis 072.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 20.</span> -- *Omodeoscolex abamaensis* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the clitellum male area); C) Habitat preferences; D) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; MA= male area; N= total number of specimens in the dataset.

\newpage
## ***Omodeoscolex tetraspermatheca* Decaëns & Bartz n. sp.**

## (Fig. 21)

```{r, include=FALSE}
c2<-c["Omodeoscolex tetraspermatheca",]
m2<-subset(m, Species == "Omodeoscolex tetraspermatheca")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimens; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited at CEFE.

<span style="font-variant:small-caps;">Etymology.</span> -- The species name refers to the presence of four unpaired spermathecae.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 21A,B)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores: `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.*

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- the two known specimens of *Omodeoscolex tetraspermatheca* n. sp. have been collected in a decaying trunk in a slope forest (Fig. 21C, D).

<span style="font-variant:small-caps;">Distribution.</span> -- *O. tetraspermatheca* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- *O. tetraspermatheca* is characterized by the presence of four unpaired spermathecae in VI–IX, whereas in the other known species of the genus with unpaired spermathecae there are three (*O. touroulti* n. sp. and *O. abamaensis* n. sp.) (Table 5). It also differs clarly from the two previously described species by its COI--barcode (Table 3). This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021.

\newpage
![](2_Figures/Decaëns et al Fig 21 Omodeoscolex tetraspermatheca 086.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 21.</span> -- *Omodeoscolex tetraspermathecae* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum position); B) External and ventral view of the anterior body (marks indicate the clitellum male area); C) Habitat preferences; D) Microhabitat preferences. All images from the Holotype and © T. Decaëns; CL= clitellum; MA= male area; N= total number of specimens in the dataset.

```{r, include=FALSE}
synt<-read.csv2("1_Datasets/Synthesis_Acantho.csv", h=T)
Table<-flextable(synt) 
Table <- italic(Table, j = 1)
Table <- fontsize(Table, size = 8, part = "all")
```
\newpage
<span style="font-variant:small-caps;">Table 5.</span> -- Main morpho-anatomical characteristics of the new Acanthodrilidae species from the Mitaraka range.
`r knit_print(Table)`

\newpage
::: {.center data-latex=""}
Family OCNERODRILIDAE Beddard, 1891

\vspace{12pt}
**Genus *Guianodrilus*  Bartz & Decaëns, gn. nov**
:::

\vspace{12pt}
<span style="font-variant:small-caps;">Etymology.</span> -- The name of the genus refers to the Guianan Center of endemism [@cardosodasilvaFateAmazonianAreas2005] in which is located the type locality of the type species.

<span style="font-variant:small-caps;">Type species.</span> -- *Guianodrilus corticolous* Bartz & Decaëns n. sp.

<span style="font-variant:small-caps;">Diagnosis.</span> -- Setae eight per segment, closely paired. Dorsal pores absent. Gizzard absent. Esophagus with extramural calciferous glands in segment IX. Holandric and meronephridial. Last hearts in XII. Intestine began in XIV. Male pores in XVII. Prostatic pore and tubular prostates in XVII. Bidiverticulate spermathecae in IX.

<span style="font-variant:small-caps;">Distribution.</span> -- To date, the genus is only known from the type type locality of *G. corticolous* n. sp.

\vspace{12pt}
## ***Guianodrilus corticolous* Bartz & Decaëns n. sp.**

## (Fig. 22)

```{r, include=FALSE}
c2<-c["Guianodrilus corticolous",]
m2<-subset(m, Species == "Guianodrilus corticolous")
ht<-subset(m2, Types=="Holotype")
pt<-subset(m2, Types=="Paratype")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
pt_r<-pt[,-c(1:7, 18:22)]
pt_d<-pt_r[!duplicated(pt_r), ]
pt_d<-pt_d[order(pt_d$Exact.Site.2,decreasing=F), ]
pt_d["Nb.Specimens"]<-by(pt_r, pt_r$Exact.Site.2, nrow)
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for paratypes
x1 = paste(sep = "", pt_d[1,"Sector"], ", ", pt_d[1,"Exact.Site"], ", ", pt_d[1,"Exact.Site.2"], "; latitude/longitude: ", pt_d[1,"Lat"],"/", pt_d[1,"Lon"],"; elevation: ", pt_d[1,"Elev"], " m asl; ", pt_d[1,"Collection.Date"], "; ", pt_d[1,"Collectors"], " leg. (", pt_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(pt_d)>1){
for(i in 2:nrow(pt_d)) {
  y1 = paste(sep = "", pt_d[i,"Sector"], ", ", pt_d[i,"Exact.Site"], ", ", pt_d[i,"Exact.Site.2"], "; latitude/longitude: " ,pt_d[i,"Lat"],"/",pt_d[i,"Lon"],"; elevation: ", pt_d[i,"Elev"], " m asl; ", pt_d[i,"Collection.Date"], "; ", pt_d[i,"Collectors"], " leg. (", pt_d[i,"Nb.Specimens"], " specimens); ")
  x1 = paste (x1, y1)
}
}
print(x1)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```


<span style="font-variant:small-caps;">Type material.</span> -- **Holotype. `r ht[,"Country.Ocean"]`** • `r ht[,"Life.Stage"]` specimen; `r ht[1,"Sector"]`, `r ht[,"Exact.Site"]`, `r ht[,"Exact.Site.2"]`, `r ht[,"Microhabitat"]`; latitude/longitude: `r ht[,"Lat"]`/`r ht[,"Lon"]`; elevation: `r ht[,"Elev"]` m asl; `r ht[,"Collection.Date"]`; `r ht[,"Collectors"]` leg.; BOLD Sample ID: `r ht[,"Sample.ID"]`; deposited at MNHN.

**Paratypes. `r pt[1,"Country.Ocean"]`** • `r nrow(pt)` `r pt[1,"Life.Stage"]` specimen; `r x1` BOLD Sample ID: `r pt[,"Sample.ID"]`; deposited at CEFE

<span style="font-variant:small-caps;">Etymology.</span> -- The name of the species refers to the microhabitat where the type specimens have been collected.

<span style="font-variant:small-caps;">Description</span>

*External morphology.* (Fig. 22)

Body shape `r c2[,"Shape"]`. Body pigmentation `r c2[,"Pigmentation"]`. Body length: `r c2[,"Size"]` after ethanol fixation. Body mass: `r c2[,"Weight"]`. Diameter: `r c2[,"Diam_pre"]` in the preclitellar region, `r c2[,"Diam_clit"]` in the clitellum, `r c2[,"Diam_post"]` in the postclitellar region. Number of segments: `r c2[,"Segments"]`. Prostomium of `r c2[,"Prostomium"]` type. Dorsal pores: `r c2[,"Dorsal_pores"]`. Setae `r c2[,"Setae_relation"]`. Setal arrangement aa:ab:bc:cd = `r c2[,"aa"]`:`r c2[,"ab"]`:`r c2[,"bc"]`:`r c2[,"cd"]`, dd `r c2[,"dd"]`. Clitellum in `r c2[,"Clitellum"]`. Genital markings in `r c2[,"Marks"]`. Male pores `r c2["Male_pore"]`, and ovipores `r c2[,"Female_pore"]`. Spermathecal pores `r c2[,"Spermatecal_pore"]`. Nephridial pores `r c2[,"Nephridial_pore"]`.

*Internal anatomy.* (Fig. 22)

Septa: `r c2[,"Septa"]`. Gizzard: `r c2[,"Gizzard"]`. Calciferous glands: `r c2[,"Calcíferous_glands"]`. Esophagus--intestine transition `r c2[,"Intestine"]`. Typhlosole `r c2[,"Typhlosole"]`. Hearts: `r c2[,"Hearts"]`. Excretory apparatus: one pair of `r c2[,"Nephridia"]` per segment. Prostatic glands: `r c2[,"Prostate"]`. Ovaries: `r c2[,"Ovaries"]`. Testes sacs: `r c2[,"Testae"]`. Seminal vesicles: `r c2[,"Seminal_vesicles"]`. Spermathecae: `r c2[,"Spermatheca"]`.

<span style="font-variant:small-caps;">Ecology.</span> -- the two known specimens of *G. corticolous* n. sp. have been found in decaying trunks and in a lowland forest (Fig. 23).

<span style="font-variant:small-caps;">Distribution.</span> -- *G. corticolous* n. sp. is only known from the Mitaraka Massif.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r c2[1,"OTU"]` in @maggia_at_2021. It is the first species described from its genus. We determined it to be new based on the summary of genus characters implicit in the key of @hernandez-garciaBrasilisiaGenArraia2018. *Guianodrilus* n. gen. is similar to *Enicmodrilus* (Eisen, 1900) by the absence of the gizzard, calciferous glands paired in IX, male and female reproductive system holandric and metagynous, respectively, and dorsal pores absent. The principal difference between the new genus and *Enicmodrilus* is the male: while pores for *Guianodrilus* n. gen. are in XVII, in *Enicmodrilus* they are in XVIII. Both have one pair of prostates in segments XVII (*Guianodrilus*) and XVIII (*Enicmodrilus*), but *Enicmodrilus* has small thin prostates while in *Guyanodrilus* n. gen. have large elongated tubes folded in the ventral part.

<span style="font-variant:small-caps;">Fig. 22.</span> (next page) -- *Guianodrilus corticolous* n. sp.: A) External and lateral view of the anterior body (marks indicate the clitellum and male and prostatic pore position); B) External and dorsal view of the anterior body (marks indicate the clitellum); C) External and ventral view of the anterior body (marks indicate the clitellum and male area); D) Internal view indicating the pair of prostates and male and prostatic pore; C) Internal view indicating sp. All images from the Holotype; CL= clitellum; MP= male pore; PP= prostate pore; PG= prostatic gland; SP= spermathecae; TS=testes; OV=ovaries; CG=calciferous gland.

\newpage
![](2_Figures/Decaëns et al Fig 22 Guianodrilus corticolous 089.pdf){ width=80% }

<span style="font-variant:small-caps;">Fig. 22.</span>

\newpage
![](2_Figures/Decaëns et al Fig 23 Guianodrilus corticolous 089.pdf){ width=100% }

<span style="font-variant:small-caps;">Fig. 23.</span> *Guianodrilus corticolous* n. sp.: C) Habitat preferences; D) Microhabitat preferences. N= total number of specimens in the dataset.


\newpage
# OTHER SPECIES COLLECTED

::: {.center data-latex=""}
\vspace{12pt}
Family RHINODRILIDAE
:::

## ***Nouraguesia parare* Csuzdi and Pavlícek, 2011**

```{r, include=FALSE}
m2<-subset(m, Species == "Nouraguesia parare")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` adult specimens, `r by(om,om$Life.Stage,nrow)[3]` juvenile specimen, `r by(om,om$Life.Stage,nrow)[2]` cocoons; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited as follow: two adult specimens at CEFE, the rest of the material at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Pontoscolex corethrurus* (Müller, 1856)**

```{r, include=FALSE}
m2<-subset(m, Species == "Pontoscolex corethrurus")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` adult specimens, `r by(om,om$Life.Stage,nrow)[3]` juvenile specimen, `r by(om,om$Life.Stage,nrow)[2]` cocoons; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited as follow: two adult specimens at CEFE, the rest of the material at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Pontoscolex* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#046")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` fragment; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Andiorrhinus (Turedrilus)* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#079")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` damaged adult specimen, `r by(om,om$Life.Stage,nrow)[2]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Martiodrilus* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#090")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Rhinodrilus* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#073")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

::: {.center data-latex=""}
\vspace{12pt}
Family GLOSSOSCOLECIDAE
:::

## ***Righiodrilus* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#075")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

::: {.center data-latex=""}
\vspace{12pt}
Family ACANTHODRILIDAE
:::

## ***Dichogaster andina* (Cognetti de Martiis, 1904)**

```{r, include=FALSE}
m2<-subset(m, Species == "Dichogaster andina")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` adult specimens, `r by(om,om$Life.Stage,nrow)[2]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited as follow: one adult specimens at CEFE, the rest of the material at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Dichogaster bolaui* (Michaelsen, 1891)**

```{r, include=FALSE}
m2<-subset(m, Species == "Dichogaster bolaui")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimen; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Wegeneriona* sp.**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#078")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimen; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

::: {.center data-latex=""}
\vspace{12pt}
Family OCNERODRILIDAE
:::

## ***Ocnerodrilidae* sp.1**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#045")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimen; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Ocnerodrilidae* sp.2**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#087")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimens; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

## ***Ocnerodrilidae* sp.3**

```{r, include=FALSE}
m2<-subset(m, OTU == "OTU#091")
om<-subset(m2, Types=="Other")
ls<-count(om, "Life.Stage")
om_r<-om[,-c(1:7, 18:22)]
om_d<-om_r[!duplicated(om_r), ]
om_d<-om_d[order(om_d$Exact.Site.2,decreasing=F), ]
om_d["Nb.Specimens"]<-by(om_r, om_r$Exact.Site.2, nrow)

### Create the locality list for other material
x = paste(sep = "", om_d[1,"Sector"], ", ", om_d[1,"Exact.Site"], ", ", om_d[1,"Exact.Site.2"], "; latitude/longitude: ", om_d[1,"Lat"],"/",om_d[1,"Lon"],"; elevation: ", om_d[1,"Elev"], " m asl; ", om_d[1,"Collection.Date"], "; ", om_d[1,"Collectors"], " leg. (", om_d[1,"Nb.Specimens"], " specimens); ")

if (nrow(om_d)>1){
for(i in 2:nrow(om_d)) {
  y = paste(sep = "", om_d[i,"Sector"], ", ", om_d[i,"Exact.Site"], ", ", om_d[i,"Exact.Site.2"], "; latitude/longitude: ", om_d[i,"Lat"],"/",om_d[i,"Lon"],"; elevation: ", om_d[i,"Elev"], " m asl; ", om_d[i,"Collection.Date"], "; ", om_d[i,"Collectors"], " leg. (", om_d[i,"Nb.Specimens"], " specimens); ")
  x = paste (x, y)
}
}
print(x)

### Calulate % of collected specimens per habitats and microhabitats
mic<-c(by(m2, m2$Microhabitat, nrow))
hab<-c(by(m2, m2$Habitat, nrow))
```

**Studied material. `r om[1,"Country.Ocean"]`** • `r by(om,om$Life.Stage,nrow)[1]` juvenile specimen; `r x` BOLD Sample ID: `r om[,"Sample.ID"]`; deposited at MNHN.

<span style="font-variant:small-caps;">Remarks.</span> -- This species corresponds to `r m2[1,"OTU"]` in @maggia_at_2021.

\newpage
# DISCUSSION

In this work, we identified 31 species of earthworms from the material collected during a single expedition in the Mitaraka Massif. Our analyses further predict that 36 species could compose the regional species pool. The overall rarefaction curve (Fig. 2A) shows that the sampling effort was probably sufficient to detect the majority of the species present in the study area. However, it can also be predicted that intensified sampling would necessarily lead to the discovery of some additional taxa, especially in decaying trunks and in the best drained habitats (mainly plateau and slope forests). This places Mitaraka second only to the Nouragues Reserve in terms of regional earthworm diversity, and among the most diverse sites on a worldwide scale [@decaens_dna_2016; @maggia_at_2021].

In general, the levels of local diversity (alpha diversity) observed in our study range from one to 12 species per community. This is higher than the prediction by @phillipsGlobalDistributionEarthworm2019a, but also lower than some of the values cited by @jamesCommentGlobalDistribution2021. The finding of higher alpha diversity values than predicted by @phillipsGlobalDistributionEarthworm2019a meta-analysis has already been highlighted by @maggia_at_2021, and can be explained by the added value of using DNA barcoding and qualitative sampling, compared to most of the studies on which the Phillips et al. model relies. The lower values than those reported by @jamesCommentGlobalDistribution2021 may be explained by differences in scale between some studies where both the alpha and beta components may have been integrated in species richness calculations. By opposition, sampling effort in our work was well circumscribed spatially and designed to strictly sample community--scale species diversity. Generally speaking, we found alpha diversity values of the same order of magnitude as those observed by @maggia_at_2021 at several study sites in French Guiana, and also substantially similar to what is commonly reported for temperate regions [@decaensAssemblyRulesEarthworm2008; @decaensMacroecologicalPatternsSoil2010b].

We also observe a significant turnover in the composition of the assemblages, since only four species found in Mitaraka (*N. parare*, *P. corethrurus*, *D. bolaui* and *D. andina*) seem to be shared with the Nouragues fauna [@decaens_dna_2016]. This high beta diversity is linked to a high proportion of local endemics in the species pools, which suggests that there could be a very large number of earthworm species at the scale of the whole French Guiana [@decaens_dna_2016; @maggia_at_2021]. This result is also in line with the conclusions of @phillipsGlobalDistributionEarthworm2019a who showed that despite the absence of a latitudinal gradient of alpha diversity, tropical areas are globally characterised by a high geographical turnover, and consequently by higher levels of regional diversity, than in temperate regions.

Our results also provide new insights into the distribution of earthworm diversity at the local scale. In particular, we observe differences in community diversity between the main ecosystem types sampled. The fact that plateau and slope forests host the most diverse earthworm communities can be explained by the fact that they are the least environmentally constrained habitats. In comparison, lowland forests and inselberg habitats present stronger hydric constraints (periodic flooding in the former, or on the contrary xeric conditions in the latter), which can act as environmental filters and limit the composition of the communities to the better adapted fraction of the regional species pool. We also observe that the species assemblage is dominated by pigmented forms, with a high diversity of "epigeic" species inhabiting decaying trunks and other surface micro-habitats. This trend, already highlighted by @decaens_dna_2016 for the Nouragues fauna, can be explained by the oligotrophic nature of Guyanese soils, which are not very conducive to the activity of endogeic earthworms, and which would have led to a strong adaptation for the use of above--ground sources of organic matter. In other tropical regions such as Meso--America and sub--Saharan Africa, the more nutrient--rich soils host communities largely dominated by endogeic species [@fragosoEarthwormCommunitiesTropical1992].

From a taxonomic point of view, our work represents a quite remarkable contribution to the knowledge of the diversity of earthworms in French Guiana. Indeed, the description of 18 species new to science represents a near doubling of the number of species cited for this territory, since only 24 species were previously cited in the region [@brown_minhocas_2007; @pavlicek_earthworm_2012-1]. This of course underlines the extent of the taxonomic deficit that characterises tropical earthworms, and which has been mentioned by others before us [@decaensMacroecologicalPatternsSoil2010b].

We describe nine new species of *Martiodrilus*, in a genus that previously comprised 89 species (http://taxo.drilobase.org/), *i.e.* an increase of nearly 10% in the number of known species. Beyond this re-evaluation of the number of species within this genus, our results modify the vision we had of its geographical distribution. Indeed, whereas most of the species known until now had been described from the north of the Andean Range, our work move west this diversity barycentre, suggesting the possible existence of another centre of diversity in the Oriental Guiana Shield. It will be interesting in the future to carry out a complete phylogenetic analysis of this genus, in order to highlight the evolutionary relationships between the different subgenera that compose it, and try to reconstruct its biogeographic history in the north of the South American continent.

With the exception of *Andiorrhinus*, which had 45 species before our work, the other new species described here concern genera where few species were known until now (i.e. *Atatina*, *Urobenus*, *Neogaster* and *Omodeoscolex*). @decaens_dna_2016 had already assigned some OTUs sampled in the Nouragues Nature Reserve to these same genera (i.e. one in *Atatina*, one in *Urobenus*, and seven in *Neogaster*), but their DNA barcodes suggest that they are different species from the one described here. It is therefore reasonable to assume that more species may be described in the future in these genera, which may thus appear to be much more diverse in French Guiana than originally thought. This may simply reflect the fact that these groups, which are essentially made up of small species, have attracted little attention from taxonomists to date or, alternatively, that the Guiana Shield may represent a centre of diversity for them. 

The particular case of the genera *Neogaster* and *Omodeoscolex* deserves special attention. Indeed, their phylogenetic proximity to other genera of Acanthodrilidae distributed on the South American and African continents makes them an interesting illustration of how the splitting of Gondwana landmass may have influenced macroevolutionary processes in earthworms around 100 million years ago [@csuzdiUberTaxonomischenProbleme1993; @csuzdiMonographPaleotropicalBenhamiinae2010]. Thus, species with a meronephridic excretory system (*i.e.* genera *Wegeneriona* and *Neogaster*) evolved on the South American continent and species with a holonephridic excretory system (*i.e.* genera *Wegeneriella* and *Pickfordia*) evolved in Africa. *Omodeoscolex* is an exception in this scheme, as species within this genus are the only ones with holonephridia and a Neotropical distribution. It is important to emphasise here that the three new species of *Omodeoscolex* described in this work have unpaired spermathecae, which distinguishes them from previously known species and calls into question the validity of this character as a criterion for generic distinction.

This work is the first in a series of taxonomic publications in which we aim to study all the OTUs delineated by @maggia_at_2021 in order to explore more fully the diversity of earthworms in Guyana.

# ACKNOWLEDGEMENTS
All the specimens recorded in this study were collected during the “Our Planet Reviewed” Guyane-2015 expedition in the Mitaraka range, in the core area of the French Guiana Amazonian Park, organized by the MNHN and Pro-Natura international. The expedition was funded by the European Regional Development Fund (ERDF), the conseil régional de Guyane, the conseil général de Guyane, the Direction de l’Environnement, de l’Aménagement et du Logement and by the Ministère de l’Éducation nationale, de l’Enseignement supérieur et de la Recherche. It was realized in collaboration with the Parc amazonien de Guyane. This work was also supported by the DIADEMA project funded by an ‘Investissement d’Avenir’ grant managed by the Agence Nationale de la Recherche (CEBA: ANR-10-LABX-25-01; TULIP: ANR-10-LABX-41). We thanks Olivier Pascal (Pro-Natura international), Julien Touroult (MNHN) and Christopher baraloto (ECOFOG lab) for allowing TD and EL to participate to the expedition in the Mitaraka Massif. DNA barcoding was part of the earthworm barcoding campaign developped within the International Barcode of Life program. The morpho-anatomical analyses of the samples and the redaction of the new taxa descriptions was supported by a grant of the PatriNat UMS (OFB – CNRS – MNHN: UM211048). We would like to thank Csaba Csuzdi for his exptremely valuable help in disentangling the complex case for Acanthodrilidae, Luis Manuel Hernández García for making available litterature and for fruitful discussion, and Guillaume Papuga for his help on writing the R script for generating specimen lists.

# REFERENCES

<div id="refs"></div>


